tests/subsys/usb/device/src/main.c

```diff
@@ -158,7 +158,6 @@ static void test_usb_deconfig(void)
 /* Test USB Device Cotnroller API */
 static void test_usb_dc_api(void)
 {
-	/* Control endpoins are configured */
```

Comment:
Something is wrong with this commit, it does not change any functionality in the code.

Reply:
Fixed, somehow the patch was incorrectly rebased.